terraform/modules/cluster/mig/main.tf

```diff
@@ -21,12 +21,12 @@ locals {
     var.enable_ops_agent ? [{
       type        = "shell"
       destination = "/tmp/enable_cloud_ops_agent.sh"
-      source      = "${path.module}/../../../../install_scripts/install_cloud_ops_agent.sh"
+      source      = "${path.module}/../../../../scripts/install_cloud_ops_agent.sh"
```

Comment:
why are we not using install-scripts ?

Reply:
it did not seem like a big deal to me so i changed it to have all our scripts in one place